README.md

```diff
@@ -15,27 +20,27 @@ versions of the base and distro components that compose it. However a major
 or minor bump of either component necesitates a corresponding bump in the fcos
 config version.
 
-internal/
+`internal/`
```

Comment:
It seems that changing these from bullets has removed the line-break + tab on the descriptions.

Reply:
While that wasn't intentional, I think I'm ok with how it renders.